src/api_server/src/parsed_request.rs

```diff
@@ -3,10 +3,10 @@
 
 use std::fmt::Debug;
 
-use logger::{error, info, log_enabled, Level};
```

Comment:
`logging::error` used to re-export `log::error`  (e.g. `pub use log::error;`). We could also re-export `tracing::error` but I don't like this, I find re-exports add complexity and reduce readability.

I understand your point if we wanted to change away from `tracing` but I don't think it's valuable for this PR to be designed considering it may be reverted.

I think using `tracing::error` (etc.) is the simplest and best approach (I think if we avoid this with wrapping here, the same point could be argued in many cases and not be useful).

Reply:
I don't have a super strong opinion here, but I do not understand how re-exporting adds complexity or reduces readability. Re-exporting just renames something. Also, it removes the need to "polute" Cargo.toml with both `logger` and `tracing`.

I think it's worth getting more opinions here

Reply:
Mh, I can see a few (weak) arguments for re-exporting
- It would prevent us from having to add `tracing` as a dependency to a bunch of `Cargo.toml` files (however we're in the process of merging all crates, so this will resolve itself either way)
- The diff for this commit wouldn't include changes of the type `log::error!` -> `tracing::error` (however we dont tend to fully qualify our logging calls, instead mostly opting for importing the macros at the top of the file).

Overall I think reexporting would be nicer (diff would be a bit smaller, and we avoid having to specify the dependency a ton of times), but I also don't have a strong opinion.

Reply:
I've updated it to maintain the same re-exporting pattern.